src/coreclr/jit/compiler.hpp

```diff
@@ -940,7 +981,12 @@ inline regNumber genFirstRegNumFromMaskAndToggle(regMaskTP& mask)
 
     /* Convert the mask to a register number */
 
+#ifdef TARGET_ARM64
+    regNumber regNum = (regNumber)BitScanForward(mask);
+#else
     regNumber regNum = (regNumber)BitOperations::BitScanForward(mask);
+#endif
```

Comment:
It would be good to avoid these ifdefs by introducing some forwarding `BitScanForward` in the `typedef unsigned __int64 regMaskTP` case.